samples/integration/vcpkg-all-smoke/src/main.cpp

```diff
@@ -43,9 +44,9 @@ int main()
   {
     std::cout << "Creating Keyvault Clients" << std::endl;
     // keyvault
-    KeyClient keyClient(std::getenv("AZURE_KEYVAULT_URL"), credential);
-    SecretClient secretClient(std::getenv("AZURE_KEYVAULT_URL"), credential);
-    CertificateClient certificateClient(std::getenv("AZURE_KEYVAULT_URL"), credential);
+    KeyClient keyClient(smokeUrl, credential);
+    SecretClient secretClient(smokeUrl, credential);
+    CertificateClient certificateClient(smokeUrl, credential);
```

Comment:
We are using `smokeUrl` for attestation client and for KeyVault URLs. How will this work?

Reply:
it's not supposed to work. it's supposed to test that we can instantiate the clients. not that the clients conenct to random services. this test is purely to test that our vcpkg packages don't blow each other up.

Reply:
If the sample isn't supposed to work, then it's not a "sample" (by definition, samples are examples for customers to use to determine how to use our SDKs).

More importantly: If the sample isn't supposed to work, then why are we executing the sample? Isn't it sufficient to ensure that the code compiles?

Reply:
THIS IS NOT FOR CUSTOMER USE , it's for internal vcpkg pipeline to validate that the packages load properly.

Reply:
If this is not for customer use, 
1. It should not be in a directory named "samples"
2. There should be SCREAMINGLY LARGE AND ANNOYING TEXT explaining why customers should ignore it.
3. If it's not supposed to actually do anything meaningful, we shouldn't be expecting it to execute, ever.